vietanh2810/Sorting-Visualizer-VA
# feat: adding bar representation for element and hook

## Summary

This merge request resolves issue #7 

## Changes Made

- Added bars representation
- Added pre-commit hook using husky

## Verification Steps

To verify that it is working correctly, follow these steps:

1. npm start
2. go to localhost:3000
3. you should see a lot of turquoise bars
4. for the pre-commit hooks:
5. just commit anything and it will do it jobs

## Reviewers

@vietanh2810
